tests/test_config.py

```diff
@@ -332,6 +333,42 @@ def test_ssl_config_combined(tls_certificate_key_and_chain_path: str) -> None:
     assert config.is_ssl is True
 
 
+def test_ssl_config_ca_cert(
+    tls_ca_certificate_pem_path: str,
+    tls_ca_certificate_private_key_path: str,
+) -> None:
+
+    config = Config(
+        app=asgi_app,
+        ssl_certfile=tls_ca_certificate_pem_path,
+        ssl_keyfile=tls_ca_certificate_private_key_path,
+        ssl_ca_certs=tls_ca_certificate_pem_path,
+    )
+    config.load()
+
+    assert isinstance(config.ssl, ssl.SSLContext)
+    config_ssl: ssl.SSLContext = config.ssl
+    assert len(config_ssl.get_ca_certs()) == 1
```

Comment:
This doesn't test the added feature ie more than one path/str or I'm missing something ?
Adding a cert in `test_ssl` would certainly be the place you'd do that